notebooks/official/tensorboard/tensorboard_custom_training_with_custom_container.ipynb

```diff
@@ -0,0 +1,999 @@
+{
```

Comment:
<p>Initialize the SDK:</p><h3>Initialize Vertex AI SDK for Python</h3><p>Initialize the Vertex AI SDK for Python for your project and corresponding bucket.</p><p> </p><pre class="ql-syntax" spellcheck="false">aiplatform.init(project=PROJECT_ID, location=REGION, staging_bucket=BUCKET_URI)
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>